spec/integration/an_organized_organizer_with_after_callbacks_deferred.rb

```diff
@@ -0,0 +1,125 @@
+# frozen_string_literal: true
+
+require 'spec_helper'
+
+RSpec.describe 'An organized organizer with after callbacks deferred', type: :integration do
+  let!(:test_interactor_1) do
+    build_interactor('TestInteractor1') do
+      defer_after_callbacks_when_organized
+
+      after_perform do
+        context.after_perform_1a = context.after_perform_1b + 1
+      end
+
+      after_perform do
+        context.after_perform_1b = context.after_perform_2 + 1 
+      end
+
+      def perform
+        context.perform_1 = 1
+      end
+    end
+  end
+
+  let!(:test_interactor_2) do
+    build_interactor('TestInteractor2') do
+      after_perform do
```

Comment:
I'm reviewing this spec file:

https://github.com/trainual/activeinteractor/pull/2/files#diff-f359da13cebfe3b11cc41062f3e4f1aa8faed85ba2115012a21abb53fc8b7631R5

and it's name. I also notice some of the interactors in this file don't use `defer_after_callbacks_when_organized` though?

Reply:
Correct 👍 

See my explanation below...happy to huddle and talk that through if I'm missing something though.

Reply:
Yeah, that's a fair critique of the spec name. Do you think I should rename this to `an_organized_organizer_with_some_after_callbacks_deferred`?

Reply:
I think the spec name is probably fine. Is there any way for the first spec to always have the callbacks deferred? Or are they actually testing two different things..

Just for more clarity:

Spec linked above:
1) Tests for some that are deferred and not

Spec file that I commented on `an_organized_organizer_with_after_callbacks_deferred`):
1) Tests for some that are deferred and not

Reply:
Yeah the distinction is more in that one contains just interactors whereas the other contains interactors AND an organizer. What do you think about...

`an_organizer_with_after_callbacks_deferred.rb` 
and
`an_organizer_containing_organizer_with_after_callbacks_deferred.rb`
?

Reply:
Ah that makes sense. Yes, that does sounds good 👍🏼